### PR TITLE
Add the results page.

### PR DESCRIPTION
**Pull request by hRobert12**
Right now it looks kinda bad, so I'm open to suggestions on how to improve this.

### CODE REVIEW

File: src/routes/landing.js [hunk 1]
```diff
@@ -21,6 +21,13 @@ class Landing extends React.Component {
             />
           </FullWidthContainer>
 
+          <FullWidthContainer flex={columnContainerFlex}>
+            <Button
+              title='Skip search and go to results'
```
Comment (AndyCErnst) [inline src/routes/landing.js]: lol, this makes no sense in a final version but fine for now. Actually, a comment saying this is temporary might be nice

File: src/routes/results.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+const styles = StyleSheet.create({
+  search: {
+    flex: 1,
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+    backgroundColor: '#fff',
+  },
+  textWithMargin: {
```
Comment (AndyCErnst) [inline src/routes/results.js]: It's a better practice to name your styles after the element they are connected to, not what they do.
When I get time to do the designs, I'm going to stick pretty close to this pattern http://www.seattle.gov/util/MyServices/WhereDoesItGo/GlassCeramics/BrokenGlass/index.htm 
So I'm thinking we will need a style for the result header, one for the action (eg Recycle), and one for the description text.

File: src/routes/results.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Button, StyleSheet, Text, View } from 'react-native';
+
+class Results extends React.Component {
+  render() { //I'm not sure how to get the spacings right for this. Right now it looks kinda ugly.
+    return (
+      <View
+        style={styles.search}>
+        <Text style={styles.textWithMargin}>That is a tin can!</Text>
+        <Text style={styles.textWithMargin}>Recycle it</Text>
```
Comment (AndyCErnst) [inline src/routes/results.js]: You can put the icons next to the text, I already have those in the repo. You'll probably want to group the icon and related text in a View together, with the heading text out on its own.

**Review by AndyCErnst — CHANGES_REQUESTED** [review]
Nice start. Sorry I don't have the designs out yet, use the seattle.gov websites as a reference until then.

**Comment by hRobert12** [comment]
I changed the results page to look more like the Seattle where does it go page.

File: src/assets/images.js [hunk 4]
```diff
@@ -3,7 +3,8 @@ const images = {
     gear: require('./images/gear.png'),
     magGlass: require('./images/magnifying-glass.png'),
     recycleBook: require('./images/recycle-book.png'),
-    recycleSymbol: require('./images/recycle-symbol.png')
+    recycleSymbol: require('./images/recycle-symbol.png'), //Robert: Can we have a recycle symbol green?
```
Comment (AndyCErnst) [inline src/assets/images.js]: - comments like this don't belong in the code
- I want to keep the recycle symbol a different color from the compost and trash symbols

File: src/routes/results.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+      <View
+        style={styles.results}>
+        <Text style={styles.headerUpper}>How to recycle</Text>
+        <Text style={styles.headerLower}>Beverage cans</Text>
+        <FullWidthContainer flex={containerFlex}>
+          <View style={{flexDirection: 'column'}}>
+            <Image source={images.recycleSymbol}/>
+            <Text>In recycling</Text>
+          </View>
+          {/*The fact that there is a LOT of whitespace here disturbs me*/}
```
Comment (AndyCErnst) [inline src/routes/results.js]: unhelpful comment

File: src/routes/results.js [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+        <Button
+          title='Go To Landing'
+          onPress={() => this.props.navigation.navigate('Landing')}
+        />
+      </View>
+    );
+  }
+}
+
+const containerFlex = 1/1;
```
Comment (AndyCErnst) [inline src/routes/results.js]: `1/1 is the same as 1`

File: src/routes/landing.js [hunk 7]
```diff
@@ -21,6 +21,13 @@ class Landing extends React.Component {
             />
           </FullWidthContainer>
 
+          <FullWidthContainer flex={columnContainerFlex}>
+            <Button
+              title='Skip search and go to results' //Defenetly temporary. Eventually the search page will lead to the results page.
```
Comment (AndyCErnst) [inline src/routes/landing.js]: spelling, "Definitely"
Even though it's a temporary comment, it's good to keep it professional

File: src/routes/results.js [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+import images from '../assets/images.js'
+import metrics from '../themes/metrics.js'
+
+import Navbar from '../components/Navbar.js'
+import FullWidthContainer from '../components/FullWidthContainer.js'
+import LargeText from '../components/LargeText.js'
+
+class Results extends React.Component {
+  render() {
+    return ( //The info here is hardcoded, but we will enevtually want to make the info change depanding on the item scanned.
```
Comment (AndyCErnst) [inline src/routes/results.js]: spelling

**Review by AndyCErnst — CHANGES_REQUESTED** [review]
looking better

**Comment by hRobert12** [comment]
Sorry for those comments in the code, I should have put those on the PR instead of the code. I have cleaned up those comments and fixed the spelling.

**Review by AndyCErnst — APPROVED** [review]
lgtm, nice

**Pull request merged**